src/styles/layout.sss

```diff
@@ -26,12 +26,8 @@
     cursor: pointer
     &:hover
         background: #333
-        
-.bg__image
-    background-size: contain
 
-.layout__icons
-    display: inline-block
+.layout__header__icons--size
```

Comment:
I believe this CSS is applied to icon, not icons, right?

Considering there is no `layout__header__icon` at all, and we, most likely, will not have different sizes modifier `--size` makes a little sense